<?xml version="1.0"?>
<launch>
	<node  pkg="slam_tutorials" type="keyboard.py" name= "teleop"/>
	<node  pkg="slam_tutorials" type="pub_odom" name= "pub_odom"/>
	<node  pkg="slam_tutorials" type="listener.py" name= "listener"/>
	<node  pkg="slam_tutorials" type="receiver.py" name= "receiver"/>
</launch>
